### PR TITLE
Feat : Add CodeGen for Julia (http)

### PR DESCRIPTION
**Pull request by levo-777**

### CODE REVIEW

**Comment by opxdelwin** [comment]
Hey @levo-777, nit: Could you please format this PR's title and description as mentioned [here](https://opensource.com/article/18/6/anatomy-perfect-pull-request)

**Comment by levo-777** [comment]
Feat : Add CodeGen for Julia (http) #154

File: lib/codegen/codegen.dart [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'package:apidash/codegen/julia/http.dart';
```
Comment (animator) [inline lib/codegen/codegen.dart:1]: incorrect position of import statement

**Comment by animator** [comment]
@levo-777 Can you kindly update the test cases to include example Request Models POST 4 to POST 9 which have been recently added.

**Comment by levo-777** [comment]
Yes just give me some time , i'll update it ...

**Pull request merged**